fenix/build.gradle

```diff
@@ -175,6 +175,19 @@ allprojects {
     }
 }
 
+subprojects {
+    afterEvaluate {
+        if (it.hasProperty('android')) {
```

Comment:
Because otherwise, building projects that do not have the `android` property would throw a `MissingMethodException`, as the extension does not exist in those types of Gradle projects.